daemons/ipa-otpd/passkey.c

```diff
@@ -307,7 +307,7 @@ bool is_passkey(struct otpd_queue_item *item)
 
 static json_t *ipa_passkey_to_json_array(char **ipa_passkey)
 {
-    int ret;
+    int ret = 0;
```

Comment:
Also possibly unnecessary.

Reply:
This was called out by the analyzer. We probably don't want to rely on the compiler to initialize this to zero.